<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Selecione o que você gostaria </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>de pesquisar:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pesquisarButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voltarButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>441</width>
     <height>256</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QCheckBox" name="tudoRB">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>Mostrar tudo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QRadioButton" name="nomeRB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nome do Projeto       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nomeText">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="idRB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID da Medição           </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">filtros</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idText">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="alturaRB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Altura                </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">filtros</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="alturaCB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Maior que</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Menor que</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Igual a</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="alturaText">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>cm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="larguraRB">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Largura              </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">filtros</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="larguraCB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Maior que</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Menor que</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Igual a</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="larguraText">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>cm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QRadioButton" name="comprimentoRB">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Comprimento      </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">filtros</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_15">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comprimentoCB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Maior que</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Menor que</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Igual a</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_16">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="comprimentoText">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>cm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="volumeRB">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Volume               </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">filtros</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="volumeCB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Maior que</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Menor que</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Igual a</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="volumeText">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>cm3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="dataRB">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Data</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">filtros</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="dataCB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Depois de</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Antes de</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Na data de</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDateEdit" name="data">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>d/M/yyyy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tudoRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>nomeRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alturaRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>alturaCB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tudoRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>alturaRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nomeRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>nomeText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alturaRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>alturaText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tudoRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>larguraRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tudoRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>comprimentoRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tudoRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>volumeRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tudoRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>dataRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>larguraRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>larguraCB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>larguraRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>larguraText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comprimentoRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>comprimentoCB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comprimentoRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>comprimentoText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>volumeCB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>volumeText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>data</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>dataCB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>idRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>idText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tudoRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>idRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="filtros"/>
 </buttongroups>
</ui>
